### PR TITLE
next/393/20240413/v1

### PR DESCRIPTION
**Pull request by victorjulien**
Staging:
 - #10729
 - #10809
 - #10817
 - #10819
 - #10822
 - #10823
 - #10827

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1763

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information:
 

ERROR: QA failed on SURI_TLPW2_autofp_suri_time.
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPW2_autofp_stats_chk | | |
| .uptime | 138 | 144 | 104.35% | 

Pipeline 20046

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 20052

File: src/output-json-stats.h [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* Copyright (C) 2014 Open Information Security Foundation
+/* Copyright (C) 2014-2024 Open Information Security Foundation
```
Comment (lukashino) [inline src/output-json-stats.h:1]: Can we update the copyright somewhat automatically for every file every year?

**Review by lukashino — APPROVED** [review]
lgtm

**Pull request merged**